RELEASING.adoc

```diff
@@ -0,0 +1,327 @@
+Downloading org/springframework/cloud/spring-cloud-app-broker-core/1.4.0 from libs-staging-local
+Downloading org/springframework/cloud/spring-cloud-app-broker-core/1.4.0 from libs-staging-local
+Downloading org/springframework/cloud/spring-cloud-app-broker-logging/1.4.0 from libs-staging-local
+Downloading org/springframework/cloud/spring-cloud-app-broker-logging/1.4.0 from libs-staging-local
+Downloading org/springframework/cloud/spring-cloud-app-broker-core/1.4.0 from libs-staging-local
+Downloading org/springframework/cloud/spring-cloud-app-broker-logging/1.4.0 from libs-staging-local
+Downloading org/springframework/cloud/spring-cloud-starter-app-broker/1.4.0 from libs-staging-local
+Downloading org/springframework/cloud/spring-cloud-app-broker-deployer-cloudfoundry/1.4.0 from libs-staging-local
+Downloading org/springframework/cloud/spring-cloud-app-broker-deployer/1.4.0 from libs-staging-local
+Downloading org/springframework/cloud/spring-cloud-app-broker-security-credhub/1.4.0 from libs-staging-local
+Downloading org/springframework/cloud/spring-cloud-app-broker-security-credhub/1.4.0 from libs-staging-local
+----
+
+We can go and manually verify that it contains the right values.
+
+Once we are happy with the staged release, we can go and promote that build to be released by manually executing the `promote-release` job.
+
+The task output should be something similar to:
+----
+2022-01-14 11:39:48.189  INFO 21 --- [           main] i.s.c.releasescripts.Application         : Starting Application using Java 1.8.0_292 on 738a81e4-eac7-4325-5041-b3e5d0985818 with PID 21 (/opt/concourse-release-scripts-0.3.4-SNAPSHOT.jar started by root in /tmp/build/aa87dc4e)
+2022-01-14 11:39:48.195 DEBUG 21 --- [           main] i.s.c.releasescripts.Application         : Running with Spring Boot v2.5.4, Spring v5.3.9
+2022-01-14 11:39:48.195  INFO 21 --- [           main] i.s.c.releasescripts.Application         : No active profile set, falling back to default profiles: default
+2022-01-14 11:39:49.399  INFO 21 --- [           main] i.s.c.releasescripts.Application         : Started Application in 2.012 seconds (JVM running for 2.747)
+2022-01-14 11:39:49.401 DEBUG 21 --- [           main] i.s.c.r.command.CommandProcessor         : Running command processor
+2022-01-14 11:39:49.402 DEBUG 21 --- [           main] i.s.c.r.command.CommandProcessor         : Found command io.spring.concourse.releasescripts.command.PromoteCommand
+2022-01-14 11:39:49.402 DEBUG 21 --- [           main] i.s.c.r.command.PromoteCommand           : Running 'promote' command
+2022-01-14 11:39:49.495  INFO 21 --- [           main] i.s.c.r.artifactory.ArtifactoryService   : Promoting spring-cloud-app-broker/app-broker-1.4.x-stage-release-1 to libs-release-local
+2022-01-14 11:39:55.938 DEBUG 21 --- [           main] i.s.c.r.artifactory.ArtifactoryService   : Promotion complete
+Promotion complete
+----
+
```

Comment:
This is where I merge the release tag into main or the branch so as to maintain a linear history.

Reply:
Ah, I see what I did differently this time. I will merge that manually and update this document. Thanks!

Reply:
There's an argument to be made that you shouldn't do this until it's published to maven central for GA releases. We have the ability to back out artifacts from artifactory, so we could try to reset if something fails on maven central.